### PR TITLE
Adding the translate beta samples

### PR DESCRIPTION
**Pull request by nirupa-kumar**
Added samples for v3beta1 translate : 

Listing Supported Languages
* translate_list_codes_beta
* translate_list_language_names_beta

Detect Langauge of Text
* translate_detect_language_beta

Glossaries (to customize translations)
* translate_create_glossary_beta
* translate_delete_glossary_beta
* translate_get_glossary_beta
* translate_list_glossary_beta

Translating Text
* translate_translate_text_beta
* translate_translate_text_with_glossary_beta
* translate_batch_translate_text_beta

### CODE REVIEW

**Comment by kolea2** [comment]
The formatter build is failing - please run code formatting. Instructions can be found here under 'Coding Style': https://github.com/googleapis/google-cloud-java/blob/master/CONTRIBUTING.md

**Comment by nirupa-kumar** [comment]
@kolea2 Thanks a lot for reviewing the PR. Please let me know if there is anything else.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/translate/snippets/TranslateSnippetsbeta.java [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+import com.google.cloud.translate.v3beta1.TranslateTextGlossaryConfig;
+import com.google.cloud.translate.v3beta1.TranslateTextRequest;
+import com.google.cloud.translate.v3beta1.TranslateTextResponse;
+import com.google.cloud.translate.v3beta1.TranslationServiceClient;
+import com.google.cloud.translate.v3beta1.TranslationServiceClient.ListGlossariesPagedResponse;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/** Snippets for the beta features */
+public class TranslateSnippetsbeta {
```
Comment (kolea2) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/translate/snippets/TranslateSnippetsbeta.java]: please capitalize "Beta"

File: google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class ITTranslateSnippetsbeta {
```
Comment (kolea2) [inline google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java]: same here please

File: google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+  private static final String projectId = System.getenv("PROJECT_ID");
+  private static final String glossaryName = "glossary-testing";
+
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+
+  @Before
+  public void setUp() {
```
Comment (kolea2) [inline google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java]: do you need these? I don't see bout or out used anywhere.

File: google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+    System.setOut(out);
+  }
+
+  @After
+  public void tearDown() {
+    System.setOut(null);
+  }
+
+  @Test
+  public void test1_testListSupportedLanguages() {
```
Comment (kolea2) [inline google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java]: please remove all "test_" prefixes

**Comment by kolea2** [comment]
LGTM overall, just a few minor comments and questions remaining.

File: google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java (hunk 3 above)
Comment (kolea2) [reply google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java]: If no, you can remove the variables and the `@Before` and `@After` methods.

Comment (nirupa-kumar) [reply google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java]: Removing it.

File: google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java (hunk 4 above)
Comment (nirupa-kumar) [reply google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java]: I have put that in so we can have the tests run in that particular order. First test_1 , test_2 and finally test_3. Please let me know if there is any other way in which I could refactor this.

File: google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java (hunk 2 above)
Comment (nirupa-kumar) [reply google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java]: Fixed in latest commit.

File: google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java (hunk 3 above)
Comment (nirupa-kumar) [reply google-cloud-examples/src/test/java/com/google/cloud/examples/translate/snippets/ITTranslateSnippetsbeta.java]: Fixed in latest commit.

**Pull request merged**